Sources/Noora/Utilities/Terminal.swift

```diff
@@ -105,4 +108,14 @@ public struct Terminal: Terminaling {
             return true
         }
     }
+
+    /// Returns the size of the terminal.
+    public static func size() -> (rows: Int, columns: Int)? {
```

Comment:
For public variables, I'd rather lean to using variables over tuples as it's easier to limit breaking changes with them.

In this case, it can be rewritten to something like:
```swift
public struct TerminalSize: {
  public let rows: Int
  public let columns: Int
}
```